src/EA.Weee.Web/Areas/Aatf/Mappings/ToViewModel/ManageEvidenceNoteTransfer.cs

```diff
@@ -22,22 +23,28 @@ public class ManageEvidenceNoteTransfer
 
         public SubmittedDatesFilterViewModel SubmittedDatesFilterViewModel { get; protected set; }
 
+        public IEnumerable<int> ComplianceYearList { get; protected set; }
+
+        public int SelectedComplianceYear { get; protected set; }
+
         public ManageEvidenceNoteTransfer(Guid organisationId, Guid aatfId, AatfData aatfData, List<AatfData> aatfs,
             FilterViewModel filterViewModel, RecipientWasteStatusFilterViewModel recipientWasteStatusFilterViewModel,
-            SubmittedDatesFilterViewModel submittedDatesFilterViewModel)
+            SubmittedDatesFilterViewModel submittedDatesFilterViewModel, int selectedComplianceYear)
         {
             Guard.ArgumentNotDefaultValue(() => organisationId, organisationId);
             Guard.ArgumentNotNull(() => aatfData, aatfData);
             Guard.ArgumentNotNull(() => aatfs, aatfs);
             Guard.ArgumentNotDefaultValue(() => aatfId, aatfId);
-            
+
             OrganisationId = organisationId;
             AatfData = aatfData;
             Aatfs = aatfs;
             AatfId = aatfId;
             FilterViewModel = filterViewModel;
             RecipientWasteStatusFilterViewModel = recipientWasteStatusFilterViewModel;
             SubmittedDatesFilterViewModel = submittedDatesFilterViewModel;
+            ComplianceYearList = ComplianceYearHelper.FetchCurrentComplianceYearsForEvidence(SystemTime.UtcNow);
```

Comment:
@GuidoCangelosi This needs unit tests to ensure the ComplianceYearList and SelectedComplianceYear are set as expected. Add tests to ManageEvidenceNoteViewModelMapTests